releases.yml

```diff
@@ -164,6 +164,7 @@ releases:
           metadata:
             scan_sources:
               exempted_packages:
+              - redhat-release
```

Comment:
We probably want to change line 163 to be generic,
and add a short reason to each package line for context 
like `# documents rhel release notes and concerns`